src/stream_to.cxx

```diff
@@ -34,6 +34,15 @@ void begin_copy(
         "COPY "sv, table, "("sv, columns, ") FROM STDIN"sv));
 }
 
+void begin_binary_copy(
+  pqxx::transaction_base &tx, std::string_view table, std::string_view columns)
+{
+  tx.exec0(
+    std::empty(columns) ?
+      pqxx::internal::concat("COPY "sv, table, " FROM STDIN WITH BINARY"sv) :
+      pqxx::internal::concat(
+        "COPY "sv, table, "("sv, columns, ") FROM STDIN WITH BINARY"sv));
```

Comment:
Wasn't this supposed to say _`FORMAT`_ `BINARY`?

Reply:
yes, I see this used older syntax but still supported.